.azure-pipelines/ci-build-production.yml

```diff
@@ -0,0 +1,90 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+
+pr: none
+
+
+stages:
+- stage: "Test"
+  jobs:
+  - job:
+    pool:
+      vmImage: windows-latest
```

Comment:
I think that being on windows was a requirement for test reports a while back but I believe it's not the case anymore. Maybe worth trying again with ubuntu to get the performance benefits.

Reply:
The security tasks use a window image

Reply:
ah right, I forgot about that... thanks for pointing it out.